### PR TITLE
Refine wording about severity in alert-rules.md

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Refine wording in alert-rules.md

+ `重要级别`  --> `严重级别` (because `重要级别` understates the severity)
+ Add a table at the beginning of the doc to summarize the severity descriptions (a common practice in other tech docs)
+ Remove duplicate descriptions under `###` titles (duplicated 4 times)

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: alert-rules.md [hunk 1]
```diff
@@ -8,14 +8,20 @@ aliases: ['/docs-cn/dev/alert-rules/','/docs-cn/dev/reference/alert-rules/']
 
 本文介绍了 TiDB 集群中各组件的报警规则，包括 TiDB、TiKV、PD、TiDB Binlog、Node_exporter 和 Blackbox_exporter 的各报警项的规则描述及处理方法。
 
+按照严重程度由高到低，报警项可分为紧急级别 \> 严重级别 \> 警告级别三类。该分级适用于以下各组件的报警项。
+
+|  严重程度 |  说明   |
+| :-------- | :----- |
+|  紧急级别  | 最高严重程度，通常由于服务停止或节点故障导致，此时需要**马上进行人工干预**。|
```
Comment (King-Dylan) [inline alert-rules.md:15]: 这样的描述是不是会好些：
紧急级别	最高严重程度，服务不可用，通常由于服务停止或节点故障导致，此时需要马上进行人工干预。
严重级别	服务可用性下降，需要用户密切关注采取措施防止可用性下降。
警告级别	对某一问题或错误的提醒。

Comment (TomShawn) [reply alert-rules.md]: 已改

File: alert-rules.md [hunk 2]
```diff
@@ -8,14 +8,20 @@ aliases: ['/docs-cn/dev/alert-rules/','/docs-cn/dev/reference/alert-rules/']
 
 本文介绍了 TiDB 集群中各组件的报警规则，包括 TiDB、TiKV、PD、TiDB Binlog、Node_exporter 和 Blackbox_exporter 的各报警项的规则描述及处理方法。
 
+按照严重程度由高到低，报警项可分为紧急级别 \> 严重级别 \> 警告级别三类。该分级适用于以下各组件的报警项。
+
+|  严重程度 |  说明   |
+| :-------- | :----- |
+|  紧急级别  | 最高严重程度，服务不可用，通常由于服务停止或节点故障导致，此时需要**马上进行人工干预**。|
```
Comment (lilin90) [inline alert-rules.md:15]: Punctuation: keep the use of period consistent?

Comment (TomShawn) [reply alert-rules.md:15]: Yes.
```suggestion
|  紧急级别  | 最高严重程度，服务不可用，通常由于服务停止或节点故障导致，此时需要**马上进行人工干预** |
```

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**